sycl/test/xocc_tests/simple_tests/kernel_arguement_forwarding.cpp

```diff
@@ -19,19 +19,20 @@ int main() {
 
   Queue.submit([&](sycl::handler &cgh) {
     auto Accessor = Buffer.get_access<sycl::access_mode::write>(cgh);
-    cgh.single_task<class FirstKernel>([=]() {
-      "--optimize 2"_kernel_param;
-      // CHECK: v++ {{.*}}class_FirstKernel{{.*}} --optimize 2
-      Accessor[0] = 0;
-    });
+    cgh.single_task<class FirstKernel>(
+        sycl::xilinx::kernel_param("--optimize 2"_cstr, [=]() {
```

Comment:
```suggestion
        sycl::xilinx::kernel_param("--optimize 2"_cstr, [=] {
```

Reply:
At the end I wonder why not using just
```c++
sycl::xilinx::kernel_param<"--optimize 2">([=] {
```
that work in C++20 I think.

Reply:
This is not yet fully implemented in clang.
but it should be possible when it is.